website/src/_posts/2019-10-01-beam-2.16.0.md

```diff
@@ -0,0 +1,101 @@
+
+ * Fully support Python 3.5, 3.6, 3.7. ([BEAM-1251](https://issues.apache.org/jira/browse/BEAM-1251))
+ * Support customized container. ([BEAM-7907](https://issues.apache.org/jira/browse/BEAM-7907))
+
+
+### New Features / Improvements
+
+* A new count distinct transform based on BigQuery compatible HyperLogLog++ implementation. ([BEAM-7013](https://issues.apache.org/jira/browse/BEAM-7013))
+* Element counters in the Web UI graph representations for transforms for Python streaming jobs in Google Cloud Dataflow. ([BEAM-7045](https://issues.apache.org/jira/browse/BEAM-7045))
+* Add SetState in Python sdk. ([BEAM-7741](https://issues.apache.org/jira/browse/BEAM-7741))
+* Add hot key detection to Dataflow Runner. ([BEAM-7820](https://issues.apache.org/jira/browse/BEAM-7820))
+* Add ability to get the list of submitted jobs from gRPC JobService. ([BEAM-7927](https://issues.apache.org/jira/browse/BEAM-7927))
+* Write portable Beam application jar. ([BEAM-7966](https://issues.apache.org/jira/browse/BEAM-7966))
+* Execute portable Flink application jar. ([BEAM-7967](https://issues.apache.org/jira/browse/BEAM-7967))
+* SQL join selection should be done in planner, not in expansion to PTransform. ([BEAM-6114](https://issues.apache.org/jira/browse/BEAM-6114))
+* A Python Sink for BigQuery with File Loads in Streaming. ([BEAM-6611](https://issues.apache.org/jira/browse/BEAM-6611))
+* Python BigQuery sink should be able to handle 15TB load job quota. ([BEAM-7588](https://issues.apache.org/jira/browse/BEAM-7588))
+* Spark portable runner: reuse SDK harness. ([BEAM-7600](https://issues.apache.org/jira/browse/BEAM-7600))
+* BigQuery File Loads to work well with load job size limits. ([BEAM-7742](https://issues.apache.org/jira/browse/BEAM-7742))
+* Upgrade "com.amazonaws:amazon-kinesis-producer" to version 0.13.1. ([BEAM-7894](https://issues.apache.org/jira/browse/BEAM-7894))
+* External environment with containerized worker pool. ([BEAM-7980](https://issues.apache.org/jira/browse/BEAM-7980))
+* BEAM SDK has compatibility problems with go1.13. ([BEAM-7994](https://issues.apache.org/jira/browse/BEAM-7994))
+* Remove translation of bounded reads from portable Flink runner. ([BEAM-8010](https://issues.apache.org/jira/browse/BEAM-8010))
+* Remove workaround to run unbounded sources in portable pipelines. ([BEAM-8011](https://issues.apache.org/jira/browse/BEAM-8011))
+* Use OffsetRange as restriction for OffsetRestrictionTracker. ([BEAM-8014](https://issues.apache.org/jira/browse/BEAM-8014))
+* Get logs for SDK worker Docker containers. ([BEAM-8015](https://issues.apache.org/jira/browse/BEAM-8015))
+* PCollection boundedness is tracked and propagated in python sdk. ([BEAM-8088](https://issues.apache.org/jira/browse/BEAM-8088))
+* HllCount needs to handle null sketches properly. ([BEAM-8126](https://issues.apache.org/jira/browse/BEAM-8126))
+* Upgrade to joda time 2.10.3 to get updated TZDB. ([BEAM-8161](https://issues.apache.org/jira/browse/BEAM-8161))
```

Comment:
Suggestion, move dependency changes to a ne subsection called "### Dependency Changes"

Any python setup.py changes to highlight here?

Reply:
sg.

For history of `setup.py`, few major changes are:
funcsigs, grpcio, google-cloud-pubsub, google-cloud-bigtable and dill.

Reply:
Added both Java and Python upgrades to a new subsection. See next commit.